docs/src/tutorial-first-steps.md

```diff
@@ -200,17 +200,22 @@ Fortunately, Kani is able to report a coverage metric for each proof harness.
 Try running:
 
 ```
-cargo kani --visualize --harness verify_success
+cargo kani --coverage -Z line-coverage --harness verify_success
```

Comment:
I actually think we should remove this from our tutorial. We shouldn't be teaching unstable features in the first step tutorial.

We should have a separate section for experimental features and just refer to them here.